src/mockatoo/macro/MockMaker.hx

```diff
@@ -991,11 +991,7 @@ class MockMaker
 				case TInst(t,p): t.get();
 				case TEnum(t,p): t.get();
 				case TType(t,p): t.get();
-				case TAbstract(t,p): 
-					var id = t.get().type.getId().toComplex();
-
-					
-						null;
+				case TAbstract(t,p): null;
```

Comment:
I know it's the same as the default case. But I wanted to keep the separation from the original code.